content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -798,53 +798,100 @@ updates:
 
 ### `versioning-strategy`
 
-When {% data variables.product.prodname_dependabot %} edits a manifest file to update a version, it uses the following overall strategies:
+When {% data variables.product.prodname_dependabot %} edits a manifest file to update a version, there are several different potential _versioning-strategies_:
 
-- For apps, the version requirements are increased, for example: npm, pip and Composer.
-- For libraries, the range of versions is widened, for example: Bundler and Cargo.
+| Option | Supported by | Action |
+|--------|--------------|--------|
+| `auto` | Try to differentiate between between apps and libraries. Use `increase` for apps and `widen` for libraries.|
+| `increase`| Always increase the minimum version requirement to match the new version. If a range already exists, typically this only increases the lower bound. |
+| `increase-if-necessary` | Leave the constraint if the original constraint allows the new version, otherwise, bump the constraint. |
+| `lockfile-only` | Only create pull requests to update lockfiles. Ignore any new versions that would require package manifest changes. |
+| `widen`| Widen the allowed version requirements to include both the new and old versions, when possible. Typically this only increases the maximum allowed version requirement. |
+| N/A | Some package managers do not yet support configuring the `versioning-strategy` parameter. |
+
+Example:
+
+<table>
```

Comment:
Can we please use markdown for this table?

Reply:
The problem I ran into was I couldn't find a clean way to do nested tables with markdown... see the original place I grabbed this table from here in this post: https://github.com/dependabot/dependabot-core/issues/4979#issue-1196967453

I poked around stackoverflow and most of the answer recommended either HTML or a custom markdown flavor/extension, which probably isn't supported here.

I do think it renders better as a nested table, so I'm not sure what to do here.

Can you handle formatting it according to docs guidance? I'm not sure if you'll still want to do left-aligned for a nested table or some variation, so just do whatever docs team prefers.

Reply:
I just realized I'd opened this from my personal fork of docs 🤦 which can be painful to deal with as a maintainer even though I've given maintainers edit access, so if it's too annoying feel free to leave a comment with your update (or just ```suggestion I suppose works quick!)

Reply:
I am going to merge this great work, and will do the table conversion internally, so that we have the updates in our public-facing docs as soon as possible for our customers.